CONTRIBUTING.md

```diff
@@ -44,6 +44,33 @@ make the code easier to navigate.
 
 *(TODO: Provide some notes on how to prepare a development environment which
 includes the needed dependencies.)*
+### Note
+This project requires libwpewebkit-1.0-dev, libwpe-1.0-dev, libwpebackend-fdo-1.0-dev 
+and GTK4 for the build/run process
+
+After cloning the project from [https://github.com/Igalia/cog] do the following 
+to build and get the project running
+-cd into project
```

Comment:
Rather than using a list here (although I'm not sure if dashes transform to lists?), it would be nice to use the code markdown. See the documentation here: https://guides.github.com/features/mastering-markdown/